src/app/modals/encryptwallet/encryptwallet.component.ts

```diff
@@ -56,12 +56,12 @@ export class EncryptwalletComponent {
             // Handle error appropriately
             error => {
               this._rpc.state.set('ui:spinner', false);
-              this.flashNotification.open('Wallet failed to encrypt properly!');
+              this.flashNotification.open('Wallet failed to encrypt properly!', '', 0, 'err');
```

Comment:
0 milliseconds hmmm..

Reply:
I have passed 0 coz we have number for time duration in the flash service, it will not do anything as we are passing an extra parameter for error handling notification.